app/models/generic_object.rb

```diff
@@ -60,6 +64,7 @@ def properties=(options)
   def method_missing(method_name, *args)
     m = method_name.to_s.chomp("=")
     super unless property_defined?(m)
+    return call_automate(m, *args) if property_method_defined?(m)
```

Comment:
consider rewrite the logic. `property_defined?` already includes `property_method_defined?`. can we reduce the call?